_sections/89-future.md

```diff
@@ -1,9 +0,0 @@
----
-title: Future
```

Comment:
Maybe replace this with a link to the issue tracker and an invitation to join the conversation?